cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -147,6 +147,26 @@ func generateMountsModeInstall(mst *initramfsMountsState, recoverySystem string)
 	return nil
 }
 
+// copyNetworkConfig copies the network configuration to the target
+// directory. This is used to copy the network configuration
+// data from a real uc20 ubuntu-data partition into a ephemeral one.
+func copyNetworkConfig(src, dst string) error {
+	for _, globEx := range []string{
+		// for network configuration setup by console-conf, etc.
+		// TODO:UC20: we want some way to "try" or "verify" the network
```

Comment:
... or to have known-to-be-good network configuration for example in ubuntu-save, ...